en/mozorg/home-m24.ftl

```diff
@@ -90,26 +90,57 @@ m24-home-common-voice = { -brand-name-common-voice }
 m24-home-donate-your-voice-to-make = Donate your voice to make speech technology more inclusive and accessible for everyone.
 m24-home-donate-your-voice = Donate your voice
 
+m24-home-our-transformation = Our Transformation
+m24-home-how-do-legacy = How do legacy values collide with innovation? In this conversation, Mark Surman previews how we’re transforming { -brand-name-mozilla } for the era of AI.
+# Used as an accessible text alternative for an image
+m24-home-alt-transformation = Change Signal with Mark Surman and MBS
+# Used as an accessible text alternative for an image
+m24-home-alt-pink-cloud = Cloud with pink background
+m24-home-any-agent = Any Agent
+m24-home-stop-wiring-tools = Stop wiring tools together and babysitting brittle automations. With the { -brand-name-mozilla-ai-v2 } Agent Platform, you describe your goal, we generate adaptive AI agents that work with your tools and processes.
+
 ## Media Springboard
 
 m24-home-type = Type
+# Obsolete string (expires 2026-03-13)
 m24-home-author = Author
+m24-home-author-v2 = Author(s)
 m24-home-topic = Topic
 m24-home-intro = Intro
 m24-home-tag-article = Article
 m24-home-tag-podcast = Podcast
 m24-home-tag-video = Video
 m24-home-topic-news = News
+m24-home-topic-products = Products
 m24-home-topic-ai = Artificial Intelligence
+m24-home-topic-open-source-ai = Open Source AI
 m24-home-topic-ps = Privacy & Security
 m24-home-headline-you-ai-v2 = You, AI and the internet — what’s really going on?
-m24-home-introducing-anonym = Introducing { -brand-name-anonym }: Raising the bar for privacy-preserving digital advertising.
-m24-home-keeping-genai-technologies = Keeping GenAI technologies secure is a shared responsibility.
-m24-home-from-hollywood-to = From Hollywood to hip hop, artists are negotiating AI consent.
-m24-home-mozilla-celebrates-groundbreaking = { -brand-name-mozilla } celebrates groundbreaking creators in new documentary series “{ -brand-name-firefox } Presents.”
-m24-home-big-techs-power = Big tech’s power over language means power over people. Bridget Todd talks to language community leaders paving the way for voice AI in their own languages and dialects.
-m24-home-why-does-it = Why does it so often feel like we’re part of a mass AI experiment? What is the responsible way to test new technologies? Bridget Todd explores what it means to live with unproven AI systems that impact millions of people as they roll out across public life.
-m24-home-kids-are-growing = Kids are growing up in a very online world. What’s a concerned parent to do?
+m24-home-shake-to-summarize = Shake to Summarize in TIME’s Best Inventions of 2025
+m24-home-mozilla-welcomes-raffi = { -brand-name-mozilla } welcomes Raffi Krikorian as Chief Technology Officer
+m24-home-tech-target = Tech Target
+m24-home-mozilla-ai-ceo = { -brand-name-mozilla-ai-v2 } CEO talks open source AI advantages
+m24-home-introducing-thundermail-and = Introducing Thundermail and Thunderbird Pro
```

Comment:
@wen-2018 I failed to call out that these terms need to add a branding placeholder for each of them { -brand-name-* }